src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -2492,7 +2488,136 @@ def oil_water_surface_tension_model(p, t):
         )
 
     @staticmethod
-    def create_constant_ph_table(
+    def create_empty():
+        """
+        Creates a dummy (empty) PVT Table parameters with no valid data.
+        """
+        return PvtModelTableParametersDescription(
+            pressure_values=np.array([0.0]),
+            temperature_values=np.array([0.0]),
+            table_variables=[],
+            variable_names=[],
+            number_of_phases=-1,
+        )
+
+    def __eq__(self, other):
```

Comment:
I'd suggest to double-check if we use `__eq__` anywhere. If not, we could remove it since this seems to be copied from the original constant pvt table, right?

Simply remove it and run tests (in alfasim and alfasim-sdk), if everything passes, you're all set ;)

Reply:
For that `__eq__` issue take a look at https://www.attrs.org/en/stable/comparison.html#customization maybe we can use the same approach described there.

Reply:
Seems to fit in that case -> "This is especially useful when you have fields with objects that have atypical comparison properties. Common examples of such objects are NumPy arrays".

Reply:
I would try to define these numpy array objects like:
```
pressure_values: Numpy1DArray = attr.ib(
        . . . , eq=attr.cmp_using(eq=numpy.array_equal) 
    )
```
As described in the `attrs` docs shared by @prusse-martin.

Reply:
Did not the documented way work?

Reply:
I had understood that we would change this part if it was not used elsewhere. As the tests are being used [here](https://github.com/ESSS/alfasim-sdk/pull/266/files#diff-43acda733867ce6c06ff05a58287d81d14ec508789ddd167a8402c2e1a954876R778) is it still important to refactor it?

Reply:
But I think this method will be still needed, since there are an attribute that is a list of numpy arrays.

And we will need the loop over that list:
```
for array1, array2 in zip(self.table_variables, other.table_variables):
    if not np.array_equal(array1, array2):
        return False
```

Reply:
`numpy.array_equal` works on lists of ndarrays:
```python
>>> a = [numpy.array([1,2]), numpy.array([3,4])]
>>> b = [numpy.array([1,2]), numpy.array([3,4])]
>>> numpy.array_equal(a, b)
True
>>> b = [numpy.array([1,2]), numpy.array([3,5])]
>>> numpy.array_equal(a, b)
False
```

Reply:
Great! I will evaluate this change using the approach cited above.